web/pgadmin/utils/constants.py

```diff
@@ -126,3 +126,9 @@
 ACCESS_DENIED_MESSAGE = gettext(
     "Access denied: You’re having limited access. You’re not allowed to "
     "Rename, Delete or Create any files/folders")
+
+
+KEY_RING_SERVICE_NAME = 'pgAdmin4-KeyRing-Service-7'
```

Comment:
what is `-7`? Better name would be `pgAdmin4-Masterpass-Service`

Reply:
done